plugins/woocommerce/tests/e2e-pw/bin/test-env-setup.sh

```diff
@@ -8,7 +8,9 @@ echo -e 'Normalize permissions for wp-content directory \n'
 docker-compose -f $(wp-env install-path)/docker-compose.yml run --rm -u www-data -e HOME=/tmp tests-wordpress sh -c "chmod -c ugo+w /var/www/html/wp-config.php \
 && chmod -c ugo+w /var/www/html/wp-content \
 && chmod -c ugo+w /var/www/html/wp-content/themes \
-&& chmod -c ugo+w /var/www/html/wp-content/plugins"
+&& chmod -c ugo+w /var/www/html/wp-content/plugins \
+&& mkdir -p /var/www/html/wp-content/upgrade \
+&& chmod -c ugo+w /var/www/html/wp-content/upgrade"
```

Comment:
The `wp-content/upgrade` folder doesn't get created until the `wp plugin install ...` commands are executed later in this script. So we have to manually create this folder first in order to set its permissions.